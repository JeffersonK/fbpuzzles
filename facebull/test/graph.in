M2	C0	C40	1
M41	C1	C39	1
M62	C2	C23	1
M135	C3	C16	1
M85	C4	C34	1000
M99	C4	C92	1
M45	C5	C42	1
M103	C5	C12	1000
M129	C6	C41	1000
M49	C6	C59	1
M81	C7	C47	1
M71	C8	C76	1
M88	C9	C88	1
M102	C9	C15	1000
M98	C10	C16	1000
M83	C10	C82	1
M59	C11	C63	1
M70	C12	C84	1000
M17	C12	C13	1
M140	C13	C52	1
M73	C14	C7	1
M23	C15	C43	1000
M132	C15	C27	1
M119	C15	C39	1000
M72	C16	C49	1000
M95	C16	C18	1
M123	C17	C30	1
M111	C18	C37	1
M125	C19	C64	1
M128	C20	C77	1
M89	C21	C78	1
M94	C22	C49	1000
M58	C22	C10	1
M92	C23	C48	1000
M3	C23	C53	1
M19	C24	C43	1
M65	C24	C85	1000
M137	C25	C60	1
M15	C26	C73	1
M74	C27	C11	1
M112	C27	C21	1000
M56	C28	C57	1
M47	C29	C89	1000
M114	C29	C90	1000
M54	C29	C35	1
M46	C30	C90	1
M44	C30	C86	1000
M42	C31	C34	1
M63	C32	C51	1000
M146	C32	C4	1
M136	C33	C85	1
M51	C34	C33	1
M29	C35	C0	1
M144	C36	C99	1
M28	C37	C49	1
M12	C37	C1	1000
M18	C38	C84	1
M61	C39	C25	1
M68	C40	C67	1
M53	C41	C72	1000
M39	C41	C26	1
M122	C41	C15	1000
M21	C42	C41	1
M16	C43	C8	1
M120	C44	C22	1
M117	C45	C20	1
M143	C46	C62	1000
M150	C46	C31	1
M6	C47	C2	1000
M40	C47	C51	1
M127	C47	C61	1000
M20	C48	C37	1000
M84	C48	C70	1
M148	C49	C62	1
M60	C50	C97	1
M76	C51	C89	1
M1	C52	C79	1
M13	C53	C42	1000
M106	C53	C45	1
M149	C53	C78	1000
M82	C54	C87	1
M24	C55	C58	1
M97	C55	C90	1000
M22	C56	C16	1000
M108	C56	C66	1
M30	C57	C32	1
M31	C58	C95	1
M26	C59	C74	1
M77	C60	C88	1000
M105	C60	C55	1
M116	C61	C89	1000
M100	C61	C12	1
M34	C62	C48	1
M32	C63	C68	1
M8	C64	C72	1
M11	C65	C6	1
M86	C66	C29	1
M142	C67	C11	1000
M121	C67	C46	1
M134	C68	C3	1
M109	C69	C44	1
M145	C70	C86	1
M104	C71	C15	1
M35	C72	C96	1
M55	C73	C90	1000
M96	C73	C91	1
M9	C74	C94	1
M64	C75	C1	1
M79	C76	C14	1
M4	C77	C75	1
M75	C77	C92	1000
M78	C78	C83	1
M66	C78	C38	1000
M90	C79	C50	1
M50	C80	C41	1000
M107	C80	C5	1
M14	C81	C83	1000
M48	C81	C11	1000
M52	C81	C69	1
M91	C81	C77	1000
M27	C82	C38	1
M38	C83	C65	1
M80	C83	C34	1000
M67	C83	C53	1000
M118	C84	C98	1
M7	C85	C80	1
M113	C85	C76	1000
M141	C85	C15	1000
M37	C86	C21	1
M36	C87	C2	1
M43	C87	C15	1000
M57	C88	C17	1
M126	C89	C81	1
M10	C90	C72	1000
M110	C90	C26	1000
M138	C90	C61	1
M33	C91	C88	1000
M69	C91	C19	1
M133	C92	C24	1
M130	C93	C39	1000
M93	C93	C28	1
M147	C93	C31	1000
M5	C94	C93	1
M87	C95	C56	1
M115	C96	C9	1
M124	C96	C68	1000
M25	C97	C36	1
M131	C98	C72	1000
M101	C98	C54	1
M139	C99	C71	1
