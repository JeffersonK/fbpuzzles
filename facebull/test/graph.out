100
1 2 3 4 5 7 8 9 11 15 16 17 18 19 21 24 25 26 27 28 29 30 31 32 34 35 36 37 38 39 40 41 42 45 46 49 51 52 54 56 57 58 59 60 61 62 64 68 69 71 73 74 76 78 79 81 82 83 84 86 87 88 89 90 93 95 96 99 100 101 104 105 106 107 108 109 111 115 117 118 120 121 123 125 126 128 132 133 134 135 136 137 138 139 140 144 145 146 148 150
